src/test/resources/META-INF/mods.toml

```diff
@@ -142,5 +142,6 @@ license="LGPL v2.1"
     modId="overlay_layers_test"
 [[mods]]
     modId="entity_renderer_events_test"
-
```

Comment:
This blank line should probably continue to exist after the lines adding the test mod. Also this file has a merge conflict.